bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/MultiRowRangeFilterAdapter.java

```diff
@@ -0,0 +1,67 @@
+package com.google.cloud.bigtable.hbase.adapters.filters;
+
+import com.google.bigtable.v2.RowFilter;
+import com.google.cloud.bigtable.util.RowKeyWrapper;
+import com.google.common.collect.BoundType;
+import com.google.common.collect.ImmutableRangeSet;
+import com.google.common.collect.ImmutableRangeSet.Builder;
+import com.google.common.collect.Range;
+import com.google.common.collect.RangeSet;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import org.apache.hadoop.hbase.HConstants;
+import org.apache.hadoop.hbase.filter.MultiRowRangeFilter;
+import org.apache.hadoop.hbase.filter.MultiRowRangeFilter.RowRange;
+
+/**
+ * Adapter for {@link MultiRowRangeFilter}, it converts the filter into an index scan hint
+ */
+public class MultiRowRangeFilterAdapter extends TypedFilterAdapterBase<MultiRowRangeFilter>  {
+  @Override
+  public RowFilter adapt(FilterAdapterContext context, MultiRowRangeFilter filter)
+      throws IOException {
+
+    return RowFilter.newBuilder()
```

Comment:
What happen in cases where scan hints won't be invoked, like in a must pass one filter list?